tracer/build/_build/UpdateVendors/VendoredDependency.cs

```diff
@@ -148,7 +148,18 @@ private static void RewriteCsFileWithStandardTransform(string filePath, string o
                             // dnlib's only targets net461 and netstandard2.0. 
                             // For our needs, it's more correct to consider `NETSTANDARD` as 'everything not .NET Framework'
                             builder.Replace("#if NETSTANDARD", "#if !NETFRAMEWORK");
+                            
+                            // Make certain classes partial so we can extend them.
+                            foreach (var className in new[] { "SymbolReaderImpl", "PdbReader", "PortablePdbReader" })
+                            {
+                                builder.Replace($"class {className}", $"partial class {className}");
+                            }
                         }
+
+                        // Debugger.Break() is a dangerous method that may crash the process. We don't
+                        // want to take any risk of calling it, ever.
+                        builder.Replace("Debugger.Break();", @"Debug.Fail(""This shouldn't happen."");");
```

Comment:
Should we remove it completely then? 🤔 I don't really see the value in adding the Debug.Fail?

Reply:
Fair point. We can't easily remove it completely as it may be used in places where [a statement is expected](https://github.com/DataDog/dd-trace-dotnet/blob/514dac86eb87d144c5052c56f71788babe5ee457/tracer/src/Datadog.Trace/Vendors/dnlib/DotNet/Pdb/Managed/DbiScope.cs#L151), so I've replaced it with a noop in 9ae6d5f6136a8c73f87f76e835ae6eeb2f08101a.